app/electron/main.ts

```diff
@@ -319,6 +344,30 @@ function getDefaultAppMenu(): AppMenu[] {
           role: 'forcereload',
           id: 'original-force-reload',
         },
+        {
```

Comment:
The new menus items should be in a new top menu "Navigate".
We should move the Reload there too. So in the end we get:
```
Navigate
  Reload
  separator
  Go to Home
  Go Back
  Go Forward
```

Reply:
done